microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Data and privacy in WebView2
+description: WebView2 data collection and practices, privacy, and the 'Diagnostic data' setting.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data and privacy in WebView2
+
+The WebView2 team tracks data to keep WebView2 secure and up to date, diagnose issues, and improve WebView2. This article is for WebView2 developers.
+WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License. Additionally, WebView2 follows the standards outlined in the [Microsoft Edge Privacy Whitepaper](../../privacy-whitepaper/index.md).
+
+In general, WebView2 data collection goes through the same strict standards as Microsoft Edge data collection. The main telemetry sources are Chromium and Microsoft Edge telemetry infrastructure, Windows data reporting, and Watson (Microsoft Internal) infrastructure to collect crash dumps.
+Any WebView2 data collection complies with the laws, regulations, and best practices for privacy, and WebView2 has guardrails to protect against privacy issues.

```

Comment:
```suggestion
Any WebView2 data collection complies with the laws, regulations, and best practices for privacy, and WebView2 has mechanisms to ensure privacy.
```
improve wording at:  has guardrails to protect against privacy issues
*  guardrails = colloquialism; use more direct technical term.
*  try a consistently positive wording; "protecting against privacy" sounds opposite, like double-negative.